Wikipedia/Code/Article as a Living Document/ArticleAsLivingDocLargeEventCollectionViewCell.swift

```diff
@@ -294,6 +327,14 @@ class ArticleAsLivingDocLargeEventCollectionViewCell: CollectionViewCell {
         setNeedsLayout()
     }
 
+    @objc private func thankButtonTapped() {
```

Comment:
The `Thanks` don't appear to persist between sessions, returning to the app later – is that expected?

<img width="770" alt="persistence" src="https://user-images.githubusercontent.com/5652327/96520599-48891c80-1224-11eb-89d3-0b5dc28f415b.png">

Reply:
That's the behavior on the existing history page, so I replicated that here. It seems weird. (And probably you can only actually thank once via the API, but it silently fails and lets you pretend you thanked them again?)

Reply:
Yeah, history doesn't have it either. I'm pretty sure it was due to an API limitation ([here's](https://www.mediawiki.org/wiki/Extension:Thanks#API_documentation) the documentation, nothing about getting already-thanked status). We can confirm that this is okay planning today.